### PR TITLE
CRM-13584: localization of the CiviCRM installer

### PR DESCRIPTION
**Pull request by mlutfy**
- displays the CiviCRM installer in the language of the user (using seedLanguage)
- sets the default language of CiviCRM to the seedLanguage, once the installation is finished (Drupal and Drupal6, but not Wordpress, since Wordpress is not bootstrapped post-install, so we can't use the API).

Work done with @monishdeb.

### CODE REVIEW

**Comment by mlutfy** [comment]
Seems like the merge-build failed for no specific reason?

**Comment by davecivicrm** [comment]
ok to test

**Comment by davecivicrm** [comment]
Mathieu - not sure what happened. Retesting now.

File: install/index.php [hunk 1]
```diff
@@ -66,11 +68,16 @@
...
 }
 
-set_include_path(get_include_path() . PATH_SEPARATOR . $crmPath);
+$pkgPath = $crmPath . DIRECTORY_SEPARATOR . 'packages';
+
+set_include_path(
+  $crmPath . PATH_SEPARATOR .
+  $pkgPath . PATH_SEPARATOR .
+  get_include_path()
+);
```
Comment (totten) [inline install/index.php]: Not a blocker, just an aside... ClassLoader's register() function also updates the include-path, so it seems like we're doing it twice. Any idea why these lines are needed? (One theory: the change was originally written on an older version of ClassLoader?)

Comment (mlutfy) [inline install/index.php]: @totten: it's possible. The patch was initially done against 4.5-alpha. I tested without the set_include_path() and it also seems to work. Can I PR that separately?

Comment (totten) [inline install/index.php]: Yeah. IMHO, it's easier to update an existing PR, but to each his own.

**Comment by mlutfy** [comment]
As discussed, removed the set_include_path().

**Comment by totten** [comment]
Merged. Also, I added 5f40a8781bd to address a warning.

I think there may still be a half-dozen strings that aren't localized. If you search install/index.php for "<li>",you should see what I mean. However, that doesn't need to block the merge.

**Pull request merged**